packages/owl-bot/src/owl-bot.ts

```diff
@@ -88,7 +88,15 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       );
       return;
     }
-
+    logger.metric('owlbot.run_label', {
+      head,
+      base,
```

Comment:
Added debug line so that I can see why the heck we're getting `Not Found` for `.OwlBot.lock` so frequently in logs.